src/main/java/sirius/db/mongo/MongoQuery.java

```diff
@@ -331,7 +331,7 @@ public Value aggregateMin(@Nonnull Mapping field) {
 
     @Override
     public void delete(@Nullable Consumer<E> entityCallback) {
-        iterateAll(entity -> {
+        streamBlockwise().forEach(entity -> {
```

Comment:
**Dangerous**: mango do not support streamBlockwise with sorting! This makes this PR breaking as all direct calls to `.delete()` must be reviewed!